src/main/java/org/spongepowered/api/Server.java

```diff
@@ -168,7 +168,7 @@
      *
      * @param message The message to send
      */
-    void broadcastMessage(Message message);
+    void broadcastMessage(Text message);
 
     /**
      * Gets the bound {@link InetSocketAddress} this server is accepting connections from.
```

Comment:
Outside scope, but never end a sentence with 'from'. `Gets the bound {@link InetSocketAddress} from where this server is accepting connections.`